deps/uv/include/uv-private/uv-win.h

```diff
@@ -474,10 +474,7 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
   /* Used in fast mode */                                                     \
   SOCKET peer_socket;                                                         \
   AFD_POLL_INFO afd_poll_info_1;                                              \
-  union {                                                                     \
-    AFD_POLL_INFO* afd_poll_info_ptr;                                         \
-    AFD_POLL_INFO afd_poll_info;                                              \
-  } afd_poll_info_2;                                                          \
+  AFD_POLL_INFO afd_poll_info_2;                                              \
```

Comment:
I just want to make sure we acknowledge that this doesn't change the ABI because sizeof(AFD_POLL_INFO) >= sizeof(AFD_POLL_INFO*).